docs/source/assets/css/vendor/scooter.scss

```diff
@@ -294,6 +294,9 @@ a,
   border: 1px solid currentColor;
   border-radius: 3px;
   cursor: pointer; }
+  .c-btn:not(button, [type=submit]) {
+    /*! You must ensure the element matches one of these selectors: button, [type=submit] */
+    outline: 2px solid red !important; }
   .c-btn[disabled], .c-btn.c-btn--disabled {
```

Comment:
When do you have a button with `.c-btn--disabled` that doesn't also have the disabled HTML property? Same question for input fields further down in the file.